frontend/components/SQLEditor/theme.css

```diff
@@ -29,7 +29,7 @@
 }
 
 .ace-fleet .ace_content {
-  height: 100% !important;
```

Comment:
This has been here from the very beginning.  The editor sets a height on the content programmatically, which this has been overriding.  If anyone knows a case where this was necessary, let me know.